build/.goreleaser.yml

```
@@ -30,55 +30,47 @@ builds:
-  - id: nri-deb
-    builds:
-      - nri-nix
+  - id: linux
     file_name_template: "{{ .ProjectName }}_{{ .Version }}-1_{{ .Arch }}"
     vendor: "New Relic, Inc."
     homepage: "https://www.newrelic.com/infrastructure"
     maintainer: "New Relic Infrastructure Team <infrastructure-eng@newrelic.com>"
     description: "New Relic Infrastructure redis Integration extend the core New Relic\nInfrastructure agent's capabilities to allow you to collect metric and\nlive state data from redis components."
     license: "https://newrelic.com/terms (also see LICENSE installed with this package)"
-    # Formats to be generated.
-    formats:
-      - deb
-    bindir: "/var/db/newrelic-infra/newrelic-integrations/bin"
-    files:
-      "CHANGELOG.md": "/usr/share/doc/nri-redis/CHANGELOG.md"
-      "README.md":  "/usr/share/doc/nri-redis/README.md"
-      "LICENSE": "/usr/share/doc/nri-redis/LICENSE"
-    config_files:
-      "redis-definition.yml": "/var/db/newrelic-infra/newrelic-integrations/redis-definition.yml"
-      "redis-config.yml.sample": "/etc/newrelic-infra/integrations.d/redis-config.yml.sample"
-    dependencies:
-      - "newrelic-infra"
 
-  - id: nri-rpm
     builds:
       - nri-nix
-    file_name_template: "{{ .ProjectName }}-{{ .Version }}-{{ .Arch }}"
-    vendor: "New Relic, Inc."
-    homepage: "https://www.newrelic.com/infrastructure"
-    maintainer: "New Relic Infrastructure Team <infrastructure-eng@newrelic.com>"
-    description: "New Relic Infrastructure redis Integration extend the core New Relic\nInfrastructure agent's capabilities to allow you to collect metric and\nlive state data from redis components."
-    license: "https://newrelic.com/terms (also see LICENSE installed with this package)"
-    # Formats to be generated.
-    formats:
-      - rpm
-    bindir: "/var/db/newrelic-infra/newrelic-integrations/bin"
-    files:
-      "CHANGELOG.md": "/usr/share/doc/nri-redis/CHANGELOG.md"
-      "README.md":  "/usr/share/doc/nri-redis/README.md"
-      "LICENSE": "/usr/share/doc/nri-redis/LICENSE"
-    config_files:
-      "redis-definition.yml": "/var/db/newrelic-infra/newrelic-integrations/redis-definition.yml"
-      "redis-config.yml.sample": "/etc/newrelic-infra/integrations.d/redis-config.yml.sample"
+
     dependencies:
       - "newrelic-infra"
+
+    bindir: "/var/db/newrelic-infra/newrelic-integrations/bin"
+
+    contents:
+      - src: "redis-config.yml.sample"
+        dst: "/etc/newrelic-infra/integrations.d/redis-config.yml.sample"
+      - src: "CHANGELOG.md"
+        dst: "/usr/share/doc/nri-redis/CHANGELOG.md"
+      - src: "README.md"
+        dst: "/usr/share/doc/nri-redis/README.md"
+      - src: "LICENSE"
+        dst: "/usr/share/doc/nri-redis/LICENSE"
+      - src: "redis-definition.yml"
+        dst: "/var/db/newrelic-infra/newrelic-integrations/redis-definition.yml"
+        type: config
+
     overrides:
       rpm:
+        file_name_template: "{{ .ProjectName }}-{{ .Version }}-{{ .Arch }}"
```

If we do this we can simplify the per-arch replacements
```suggestion
        file_name_template: "{{ .ProjectName }}-{{ .Version }}-1.{{ .Arch }}"
```